templates/zerver/help/link-to-a-message-or-conversation.md

```diff
@@ -1,10 +1,29 @@
 # Link to a message or conversation
 
-Share permanent links to messages, threads, and streams.  With Zulip,
-it's easy to to link to specific (parts of) conversations from issue
-trackers, documentation, or other external tools.
+Share permanent links to messages, topics, and streams. With Zulip,
+it's easy to to link to specific (parts of) conversations from
+issue trackers, documentation, or other external tools.
 
-## Link to a stream or topic
+Also, when composing a message in Zulip, you can use
+[Zulip's Markdown][zulip-markdown-links]
+features to easliy auto-link to existing streams and topics.
+
+[zulip-markdown-links]: /help/format-your-message-using-markdown#links
+
+## Markdown syntax to auto-link to a stream or topic
+```
+Stream: #**stream name**
+Topic: #**stream name>topic name**
+```
+
+!!! tip ""
+
+    When using this Markdown feature in the compose box, a menu of
+    valid streams should appear as you're typing. Once you've highlighted
+    the desired stream name from the menu, you can press `>` for a list
+    of valid topic names in that particular stream.
+
+## Permanent link to a stream or topic
```

Comment:
I think this should just be "Link to..." -- the links this generates are permanent for a stream but not for a topic.

Reply:
If that's the case, then I think we should also adjust the first sentence of the page as well for consistency ...
"Share links to messages, topics, and streams." vs "Share permanent links to messages, topics, and streams."

The description text for links to specific messages explicitly says that it generates a permanent link.